eli5/sklearn/explain_prediction.py

```diff
@@ -198,11 +201,18 @@ def explain_prediction_linear_classifier(clf, doc,
             add_weighted_spans(doc, vec, vectorized, target_expl)
             res.targets.append(target_expl)
     else:
+        if len(display_names) == 1:  # target is passed explicitly
+            label_id, target = display_names[0]
+        else:
+            label_id = 1 if score >= 0 else 0
+            target = display_names[label_id][1]
+        scale = -1 if label_id == 0 else 1
+
         target_expl = TargetExplanation(
-            target=display_names[1][1],
-            feature_weights=_weights(0),
+            target=target,
+            feature_weights=_weights(0, scale=scale),
             score=score,
```

Comment:
Just to clarify - we want to invert the weights and show proba for the predicted class, and show the score unchanged, right? So for negative prediction we should get say "y=0 proba 0.8 score -10", right?